### DISCUSSION TITLE
Azure B2C - czy jest możliwość dodania custom domain

### DISCUSSION THREAD

**Discussion by dudus999** [AzureCommunityPL/group#224] [Pomoc]
Cześć,

Wydaje mi się, że się nie da ale zapytam:

Czy jest jakiś sposób aby zrobić inny link do żądań do b2c 
Czyli domyślnie mamy
https://tenantname.b2clogin.com/tenantname.onmicrosoft.com 
Czy istnieje możliwość zmiany nazwy bez rejestracji nowego tenanta
https://nowanazwa.b2clogin.com/nowanazwa.onmicrosoft.com ?

**Comment by rafal-linowiecki** [comment]
Z mojego doświadczenia nie można zmienić nazwy tenanta - jedynie jak piszesz - rejestrujesz nowego. Masz jeszcze opcje podpięcia custom domain ale to trochę inne zagadnienie

**Comment by PawelHaracz** [comment]
W tej chwili nie ma takiej możliwości.
Ale warto obserwować ten user voice - zaczęli nad tym pracować

https://feedback.azure.com/forums/169401-azure-active-directory/suggestions/15334317-customer-owned-domains

**Reply by PawelHaracz** [reply]
@dudus999  wlasnie dzisiaj ogłosili, jest narazie w preview, ale może to Ci pomoże
https://docs.microsoft.com/en-us/azure/active-directory-b2c/custom-domain?branch=main&pivots=b2c-custom-policy

**Reply by wojtekdo** [reply]
To ja dodam jeszcze, że działa spoko. Można dorzucić wiele domen, a jako że całość leci przez Front Door to jest pole do eksperymentowania co tam ciekawego da się wycisnąć poza samą podmianą domeny.

**Reply by dudus999** [reply]
U mnie też już działa.
